openedx/features/course_experience/__init__.py

```diff
@@ -72,7 +72,8 @@
 # .. toggle_creation_date: 2017-09-11
 # .. toggle_target_removal_date: None
 # .. toggle_warnings: This is meant to be configured using waffle_utils course override only. Either do not create the
-#   actual waffle flag, or be sure to unset the flag even for Superusers.
+#   actual waffle flag, or be sure to unset the flag even for Superusers. This is no longer used in the learning MFE
+#   and can be removed when the outline tab is fully moved to the learning MFE.
```

Comment:
This flag is still checked in the old LMS outline tab:
https://github.com/edx/edx-platform/blob/master/openedx/features/course_experience/views/course_home.py#L148

I figured I'd leave it alone for now?

Reply:
Do we care though? Doesn't your new code just always retrieve the latest update? I feel like we could potentially just get rid of it there too and have the course_home.py use your same code path. I also understand leaving it alone. Might be good to not break parity at the moment.

Reply:
Ah! That was part of my initial confusion too before this refactor. The flag code path (the "latest update") grabbed the same data (the latest update) -- it just renders it differently. User can't permanently dismiss, only temporary dismiss; and it's labelled as "Latest Update" instead of the normal labelling.

The flag is only enabled for like eight 2017 MITxPro courses.

Reply:
I vote kill it with fire then?

Reply:
Eh, it would probably need to do a DEPR and all that, since open edx might be using it.

Reply:
You're too kind, but I understand. My perspective is that it pulls the same code regardless, just a slightly different UI change. This is a feature improvement more than a deprecation. But I trust your instincts as well.